src/tests/ftest/network/cart_self_test.py

```diff
@@ -104,21 +100,30 @@ def setUp(self):
         # Start the daos server
         self.start_server_managers()
 
+        # Generate a uri file using daos_agent dump-attachinfo
+        attachinfo_file = "{}.attach_info_tmp".format(self.server_group)
+        self.uri_file = os.path.join(self.tmp, attachinfo_file)
+        agent_cmd = self.agent_managers[0].daos_agent
```

Comment:
There already exists a property for this object to reference the `DaosAgentCommand` object:
```suggestion
        agent_cmd = self.agent_managers[0].job
```

Reply:
Ah, cool. Didn't pick up on that. Fixed.

Reply:
hmmm...
```
  File "/home/mjmac/daos/install/lib/daos/TESTING/ftest/network/cart_self_test.py", line 107, in setUp
    agent_cmd = self.agent_managers[0].job
AttributeError: 'DaosAgentManager' object has no attribute 'job'
```

Reply:
Ah, got it. `self.agent_managers[0].manager.job`